File: python/easy/0268_missing_number.py
# https://leetcode.com/problems/missing-number/description/
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        expectedSum = 0
        actualSum = 0
        for i in range(len(nums)):
            expectedSum += i
            actualSum += nums[i]
        expectedSum += len(nums)
        return expectedSum - actualSum

File: cpp/easy/0268_missing_number.cpp
// https://leetcode.com/problems/missing-number/description/
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int expectedSum = 0;
        int actualSum = 0;

        for(int i = 0; i < nums.size(); i++){
            expectedSum += i;
            actualSum += nums[i];
        }
        return expectedSum + nums.size() - actualSum;
    }
};
